src/Conversion/KrnlToAffine/KrnlToAffine.cpp

```diff
@@ -975,6 +973,9 @@ class KrnlMatmulLowering : public OpRewritePattern<KrnlMatMulOp> {
     // can simdize only if K is compile time
     assert(J.isLiteral() &&
            "can only simdize with compile time blocking factor on simd axis");
+    MultiDialectBuilder<AffineBuilderKrnlMem, MemRefBuilder, KrnlBuilder>
```

Comment:
Is the new builder used in the code ? Could not find it.

Reply:
your right, I updated the code to support MultiDialectBuilder fo VectorBuilder. Code is uploaded already.